api/src/main/templates/LanguageCode.java.tpl

```diff
@@ -279,7 +320,9 @@ declaration += ")"
         else
             result = iso639_3_index.get(iso639);
         if (result == null)
-            throw new IllegalArgumentException("Invalid ISO639 " + iso639);
+            result = iso639_2b_index.get(iso639);
+            if (result == null)
```

Comment:
Maybe I'm missing something here-- should this not be 4 spaces in from the outer `if` level like everything else in the section?

Reply:
If you mean the `if` on line 322, that is not an outer `if`, because there are no braces after it. That is, the `if` on line 322 only governs the statement on line 323.

Reply:
oh got it, I figured this was some sort of weird template-specific "we don't believe in curly braces" situation but stackexchange has just informed me that's actual java syntax I've just never seen before. you learn something new every day!